app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -12,7 +12,7 @@ CONFIG_COMP_MULTIBAND_DRC=y
 CONFIG_PM=y
```

Comment:
I'm having a bit hard time to understand the commit:
```
Since D3 is system power-off, all devices are restarted and we don't
need to do any IPC suspend. SOF already check conditions for successful
shutdown and if everything is OK we can just go to power-off in idle.
```

Is this talking about case of IMR context-save case or regular D3? I'm guess in regular case, suspend is not needed as we'll start from scratch anyways. But in context-save case,  why do we actually have to suspend? The state of the IPC driver should be restored from IMR, so it should just continue from whatever state it was before state save. OTOH, if there's a reason, then this patch seems to be the way to do it, but the commit is not very easy to follow.